packages/salesforcedx-vscode-core/test/vscode-integration/util/authInfo.test.ts

```diff
@@ -79,15 +88,14 @@ describe('OrgAuthInfo', () => {
         true
       );
       expect(showMessageStub.calledOnce).to.equal(true);
-
-      configUtilStub.restore();
-      showMessageStub.restore();
-      executeCommandStub.restore();
     });
 
     it('should not show a message if there is a dev hub set', async () => {
-      const configUtilStub = sandbox.stub(ConfigUtil, 'getConfigValue');
-      configUtilStub.returns('username');
+      const configAggregatorStub = sandbox.stub(
+        ConfigAggregator.prototype,
+        'getPropertyValue'
+      );
+      configAggregatorStub.returns('username');
```

Comment:
nit: totally a best practices thing, but if I'm creating a stub in a test I always have an assertion to verify it was called and potentially an assertion to verify the args passed.